### PR TITLE
Working Mode: clarify that major changes warrant additional time

### LINKED ISSUE #98: Working Mode: clarify that "Changes" is asynchronous and subject to appeal
https://whatwg.org/working-mode#changes is not as clear as it could be that major changes are best not announced and committed to master within a day or so. Some time should pass for review and appeal to ensure the broadest possible buy-in while still moving relatively quickly.

Maybe a third bullet point could be added to the second list saying something to the effect of:

* For major changes enough time should have passed for review and appeal by the larger community. Generally a full week is considered sufficient.

(Generally I measure this from the time an issue or PR is opened to the time where the PR could land on master.)

### PR DESCRIPTION
**Pull request by annevk**
Closes #98.

### CODE REVIEW

**Comment by travisleithead** [comment]
With my community participant hat on, even with 7 days of delay before merging, I might still miss a major change unless there was some sort of alert that I should go take a look, with a summary of what major thing was changing. I don't think there's anything that suggests this be done--is it something we should consider?

**Comment by annevk** [comment]
I think this is mainly for those that actively participate in the development. Note that it also does not account for vacation and such. And it's still a judgment call. E.g., if the editor determines the relevant people have chimed in, merging earlier can be okay. Furthermore, I'd still expect decisions to be subject to appeal if there's new relevant information.

**Comment by travisleithead** [comment]
> Furthermore, I'd still expect decisions to be subject to appeal if there's new relevant information.

The wording makes it seem like appeals are only limited to the window of 7 days.

I'm also concerned about the suggestion of 7 days becoming a rule; wondering if we can make this wording more vague.

File: Working Mode.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ Each normative change made to the standard needs to meet the following criteria:
 * It should have corresponding test changes, either in the form of new tests or modifications to existing tests.
 * Implementations bugs must be filed for each user agent that fails tests. (This is each user agent that doesn’t match the proposed changes. If the test changes are not adequate to reveal that, but it’s known through other means, the tests should be improved first.)
 * It should have been reviewed by one or more members of the community.
+   * For major changes enough time should have passed for review and appeal by the larger community. Generally 7 [Days](./IPR%20Policy.md#29-day) is considered sufficient.
```
Comment (annevk) [inline Working Mode.md]:
```suggestion
   * For major changes wider input should be sought and it is therefore recommended to let some more time pass for review.
```

**Comment by othermaciej** [comment]
Vagueness avoids the problem of anchoring on 7 days, which may sometimes not be enough. But is "some more time" too vague? Perhaps it should be "significantly more time"?

**Comment by annevk** [comment]
I think current practice would be "some more time" (though sometimes could be significant if review is tricky or requires many cycles) and I was mainly trying to codify that.

**Comment by foolip** [comment]
How about "some more time (typically a week or more)"? That still leaves room for less if the editor is confident nobody is going to mind.

**Comment by annevk** [comment]
I've made those changes. Let me know what you think!

File: Working Mode.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ Each normative change made to the standard needs to meet the following criteria:
 * It should have corresponding test changes, either in the form of new tests or modifications to existing tests.
 * Implementations bugs must be filed for each user agent that fails tests. (This is each user agent that doesn’t match the proposed changes. If the test changes are not adequate to reveal that, but it’s known through other means, the tests should be improved first.)
 * It should have been reviewed by one or more members of the community.
-   * For major changes enough time should have passed for review and appeal by the larger community. Generally 7 [Days](./IPR%20Policy.md#29-day) is considered sufficient.
+   * For major changes wider input should be sought and it is therefore recommended to let some more time (typically a week or more) pass for review.
```
Comment (crissy33) [inline Working Mode.md]:
```suggestion
   * For major changes wider input should be sought and it is therefore recommended to let some more time (typically a week or more) pass for review.
```

**Comment by foolip** [comment]
@dbaron @othermaciej can you also approve?

**Pull request merged**